packages/contracts-bedrock/package.json

```diff
@@ -14,7 +14,7 @@
     "build": "forge build",
     "build:go-ffi": "(cd scripts/go-ffi && go build)",
     "autogen:invariant-docs": "npx tsx scripts/autogen/generate-invariant-docs.ts",
-    "test": "pnpm build:go-ffi && forge test",
+    "test": "pnpm build:go-ffi && DEPLOY_CONFIG_PATH=deploy-config/hardhat.json forge test",
```

Comment:
We could use the cheatcode to set the env var in the testing environment but that isn't great due to previous issues with setting env vars from within foundry